Parser/spark.py

```diff
@@ -825,15 +826,15 @@ def resolve(self, list):
 
 def _dump(tokens, sets, states):
     for i in range(len(sets)):
-        print 'set', i
+        print('set', i)
         for item in sets[i]:
-            print '\t', item
+            print('\t', item)
             for (lhs, rhs), pos in states[item[0]].items:
-                print '\t\t', lhs, '::=',
-                print string.join(rhs[:pos]),
-                print '.',
-                print string.join(rhs[pos:])
+                print('\t\t', lhs, '::=', end=' ')
```

Comment:
`end=` is not supported in Python 2.5. Use the single `print()`.

Reply:
Fixed in 39e5248e39b35b9da2d57590361eacdf98c6e8e5.